### PR TITLE
disable HTTP/2 on kube-rbac-proxy container

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -73,7 +73,8 @@ spec:
             memory: 500M
       - name: kube-rbac-proxy
         image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.13.0
-        securityContext:
+         imagePullPolicy: Always
+         securityContext:
```
Comment (MatousJobanek) [inline config/manager/manager.yaml:77]: the formatting looks weird, it looks like there is one space extra
```suggestion
        imagePullPolicy: Always
        securityContext:
```

**Review by ranakan19 — APPROVED** [review]

**Review by alexeykazakov — APPROVED** [review]

**Comment by xcoulon** [comment]
waiting for http://registry.redhat.io/openshift4/ose-kube-rbac-proxy update

**Pull request merged**